protoc-artifacts/README.md

```diff
@@ -1,3 +1,14 @@
+# Special notes about building pre-compiled protoc-gen-javalite
+This section is only visible from the javalite branch, which is forked from the
+master branch to provide a stable java lite support. The ``protoc-gen-javalite``
+plugin can only be built and published in this branch.
+
+The sections below in this README are branched from the master, with all the mvn
+commands changed to have an extra ``-f javalite-pom.xml`` parameter to load the
+correct configuration for plugins. The original ``pom.xml`` in this branch is
+renamed to ``protoc-pom.xml`` to prevent the protoc from being accidentally
+published using this branch.
+
 # Build scripts that publish pre-compiled protoc artifacts
 ``protoc`` is the compiler for ``.proto`` files. It generates language bindings
```

Comment:
I'd like to keep the doc unchanged to ease the future cherry-pick/merges. I have pointed out in the 2nd section about how this README differs from the one in master.